db/fixtures/miq_policies.yml

```diff
@@ -0,0 +1,29 @@
+---
+- :name: Analyse all new container images for known vulnerabilities with OpenSCAP ruleset
```

Comment:
@moolitayer typically names are not underscored, why can't we just a short name and have a long description?  What does the underscore buy us?